gallery/workbook/Azure Monitor.json

```diff
@@ -257,13 +257,6 @@
 					"description": "Performance (query, export, ingestion, streaming) of all your Azure Data Explorer Clusters",
 					"name": "Azure Data Explorer Overview"
 				},
-				{
```

Comment:
is this not needed anymore? will you add a  new workbook here?

Reply:
yes